tests/integration_tests/build/test_binary_size.py

```diff
@@ -33,7 +34,7 @@
 
 
 @pytest.mark.timeout(500)
-def test_firecracker_binary_size():
+def test_firecracker_binary_size(record_property):
```

Comment:
So `record_property` records properties per test, right? My concern here is "recording" properties with the same key.